dotCMS/src/main/java/com/dotcms/saml/SamlException.java

```diff
@@ -0,0 +1,26 @@
+package com.dotcms.saml;
+
+/**
+ * Exception to report things related to the dot saml exception
+ * 
+ * @author jsanca
+ */
+public class SamlException extends RuntimeException
+{
+	private static final long serialVersionUID = -3569526825729783600L;
+
+	public SamlException()
```

Comment:
Codacy found an issue: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5707914)